packages/core/botpress/src/web/components/App/index.jsx

```diff
@@ -49,6 +50,10 @@ class App extends Component {
     EventBus.default.on('notifications.new', notification => {
       this.props.addNotifications([notification])
     })
+
+    new BackendToast({ title: 'nlu-sync' })
```

Comment:
Can we instead make it an (invisible) React component that we would use like `<BackendToast code="nlu-sync" />`?

Also I don't quite like we now have module-specific logic in `core`. In my understanding the `BackendToast` should be singletone ad its manager should filter and discard (as necessary) the toasts of the same "title" accordingly